core/domain/feedback_services.py

```diff
@@ -755,12 +754,13 @@ def _send_instant_emails(
         feedback_message_reference: FeedbackMessageReference.
     """
     for recipient_id in recipient_list:
-        recipient_preferences = (
-            user_services.get_email_preferences(recipient_id))
-        if recipient_preferences['can_receive_feedback_message_email']:
-            transaction_services.run_in_transaction(
-                enqueue_feedback_message_instant_email_task, recipient_id,
-                feedback_message_reference)
+        if user_services.is_user_registered(recipient_id):
```

Comment:
I'm not sure this is the correct place to do this. Can we not have an empty recipient_id in any of the recipient_lists at all?